README.rst

```diff
@@ -29,11 +30,16 @@ of critical security issues) or with an extensive deprecation period.
 Issues and requests against GA libraries are addressed with the highest
 priority.
 
+.. note::
+
+    Sub-components of GA libraries explicitly marked as beta in the
+    import path (e.g. ``google.cloud.language_v1beta2``) should be considered
+    to be beta.
```

Comment:
Do you want to add this note to `language/README.rst` as well?

Reply:
Nah. I think it is salient enough that saying it generally in one place is acceptable.